cyfronet-fid/marketplace
# [#2333] OpenAIRE integration for EGI Notebooks

Identify the service by its PID. For it to work in development handle
pid as an optional field in dev:prime.

Enable the integration for PID egi-fed.notebook.

Add the integration box, with a link leading to the predefined URL.

## How to test

Go to the service `EGI Notebook` and verify, that the information is displayed correctly.
It should be invisible for other services.
Also, it is not to be displayed in backoffice or other management paths.

Closes: #2333.
